src/pybind/ceph_daemon.py

```diff
@@ -0,0 +1,275 @@
+            pad_prefix = pad / 2
+            header += (pad_prefix * '-')
+            header += (section_name[0:section_width])
+            header += ((pad - pad_prefix) * '-')
+            header += ' '
+        header += "\n"
+        ostr.write(self.colorize(header, self.BLUE, True))
+
+        sub_header = ""
+        for section_name, names in self._stats.items():
+            for stat_name, stat_nick in names.items():
+                sub_header += self.UNDERLINE_SEQ \
+                              + self.colorize(
+                                    stat_nick.ljust(self.col_width(stat_nick)),
+                                    self.BLUE) \
+                              + ' '
+            sub_header = sub_header[0:-1] + self.colorize('|', self.BLUE)
+        sub_header += "\n"
+        ostr.write(sub_header)
+
+    def _print_vals(self, ostr, dump, last_dump):
+        """
+        Print a single row of values to `ostr`, based on deltas between `dump` and
+        `last_dump`.
+        """
+        val_row = ""
+        for section_name, names in self._stats.items():
+            for stat_name, stat_nick in names.items():
+                stat_type = self._schema[section_name][stat_name]['type']
+                if bool(stat_type & COUNTER):
+                    n = max(dump[section_name][stat_name] -
+                            last_dump[section_name][stat_name], 0)
+                elif bool(stat_type & LONG_RUNNING_AVG):
+                    entries = dump[section_name][stat_name]['avgcount'] - \
+                            last_dump[section_name][stat_name]['avgcount']
+                    if entries:
+                        n = (dump[section_name][stat_name]['sum'] -
+                             last_dump[section_name][stat_name]['sum']) \
+                            / float(entries)
+                        n *= 1000.0  # Present in milliseconds
+                    else:
+                        n = 0
+                else:
+                    n = dump[section_name][stat_name]
+
+                val_row += self.format_dimless(n, self.col_width(stat_nick))
+                val_row += " "
+            val_row = val_row[0:-1]
+            val_row += self.colorize("|", self.BLUE)
+        val_row = val_row[0:-len(self.colorize("|", self.BLUE))]
+        ostr.write("{0}\n".format(val_row))
+
+    def _load_schema(self):
+        """
+        Populate our instance-local copy of the daemon's performance counter
+        schema, and work out which stats we will display.
+        """
+        self._schema = json.loads(admin_socket(self.asok_path, ["perf", "schema"]))
+
+        # Build list of which stats we will display, based on which
+        # stats have a nickname
+        self._stats = defaultdict(dict)
+        for section_name, section_stats in self._schema.items():
+            for name, schema_data in section_stats.items():
+                if schema_data['nick']:
```

Comment:
if in doubt, there is nothing wrong in doing a `.get()` operation on a dictionary, where it will default to `None` so your `if` condition would still work